README.md

```diff
@@ -0,0 +1,129 @@
+# StackStorm in Docker containers
+
+The initial container configuration is as follows:
+
+ - st2-upstart (st2 + st2web + sshd)
+ - mongo
+ - rabbitmq
+
+## Usage
+
+Build the st2-upstart image:
+
+  ```
+  make build
+  ```
+
+Start the docker environment:
+
+  ```
+  docker-compose up
```

Comment:
Good idea. I'm going to add `-d`.

I've added `reconfigure the logger to output to the CLI` to the backlog.